packages/editor/src/panels/files/fileitem.tsx

```diff
@@ -139,11 +139,12 @@ function TableView({ file, onClick, onDoubleClick, drag, drop, isOver, onContext
       onContextMenu={onContextMenu}
       onClick={onClick}
       onDoubleClick={onDoubleClick}
+      data-testid="files-panel-file-item"
     >
       {availableTableColumns
         .filter((header) => selectedTableColumns[header])
         .map((header, idx) => (
-          <td key={idx} style={{ fontSize: `${fontSize}px` }}>
+          <td key={idx} style={{ fontSize: `${fontSize}px` }} data-testid={`files-panel-file-item-${header}`}>
```

Comment:
how about using `idx` instead of `header`?
`idx` is more non-changing

Reply:
@aditya-mitra Ah yes, that's a good point! I'll make the change.

Reply:
@aditya-mitra So for this, the reason I went with `header` is so that the `data-testid` is descriptive. If we use `idx`, we'd have to map out what each index position corresponds to -- which isn't that big of a deal, but if we can avoid that I'd prefer that. 

This is what we get when we use `idx` instead of `header`:
![image](https://github.com/user-attachments/assets/1d07d70a-978d-4673-80bf-3a7fa7727123)

As you can see in the screenshot, `files-panel-file-item-0` doesn't tell us much. However, if we use `header`:
![image](https://github.com/user-attachments/assets/df0b6e60-33ef-4865-a6ee-0939264e9be9)

Now, we know exactly the element we're getting, which in this case is the **name** of the "file item".

Reply:
awesome explanation with visuals!
got it. we can keep headers then